workers/unity/Packages/com.improbable.gdk.core/Attributes/RemoveAtEndOfTick.cs

```diff
@@ -4,11 +4,13 @@
 namespace Improbable.Gdk.Core
 {
     /// <summary>
-    ///     Any component with this attribute will be removed from all entities by the CleanReactiveComponentSystem
+    ///     Any component that inheriting this interrace will be removed from all entities
+    ///     by the CleanReactiveComponentSystem.
     ///     Can be added to components extending <see cref="IComponentData" /> or <see cref="ISharedComponentData" />
     /// </summary>
-    [AttributeUsage(AttributeTargets.Struct, AllowMultiple = false, Inherited = false)]
-    public class RemoveAtEndOfTick : Attribute
+
+    public interface RemoveAtEndOfTick
```

Comment:
nitpick - the naming convention is IRemoveAtEndOfTick. But if it's an interface instead of an attribute we might want to rename it.